internal/user/password_hashings_test.go

```diff
@@ -0,0 +1,32 @@
+package user
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kopia/kopia/internal/crypto"
+)
+
+// The password hashing constants defined in this package are used as "lookup
+// keys" for the register password-based key derivers in the crypto package.
+// This trivial test is a change detector to ensure that the constants defined
+// in the user package match those defined in the crypto package.
+func TestPasswordHashingConstantMatchCryptoPackage(t *testing.T) {
```

Comment:
@bathina2 This is what I meant by _"adding a test to ensure the constants match"_.